src/Loader/CucumberNDJsonAstLoader.php

```diff
@@ -51,7 +52,7 @@ private static function getFeature(array $json, $filePath)
 
         return new FeatureNode(
             $featureJson['name'] ?? null,
-            $featureJson['description'] ? trim($featureJson['description']) : null,
+            trim($featureJson['description'] ?? '') ?: null,
```

Comment:
note: it seems odd to me that 'name'/title of the feature can be optional, but the description is mandatory. This change relaxes the restriction and slightly improves code readability.